packages/blade/src/components/Icons/TransactionIcon/TransactionIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { Svg, Path } from '../_Svg';
+import type { IconComponent } from '..';
+import useIconProps from '../useIconProps';
+
+const TransactionIcon: IconComponent = ({ size, color }) => {
```

Comment:
In figma and products I think we call it `Transactions` (with s). Guess will be better to name it that.

Reply:
changed the name. please check.